### PR TITLE
[ENH] Add a bootstrap function for all changelog entries

### LINKED ISSUE #57: Backfill / bootstrap an initial changelog
**Is your feature request related to a problem? Please describe.**

It would be nice for repos that want to adopt `github-activity` to be able to automatically backfill a changelog file.

**Describe the solution you'd like**

Maybe an `--all` flag could make this convenient, something like this:

```bash
github-activity org/repo --auth <github-token> --since 1970-01-01 --branch main --all
```

**Describe alternatives you've considered**

An alternative could be to write a small wrapper around `github-activity` that would fetch all tags, and call `github-activity` sequentially.

Or do that by hand if it's just a couple of releases to backfill.

**Additional context**

### PR DESCRIPTION
**Pull request by blink1073**
Fixes #57 

e.g. `github-activity executablebooks/github-activity --all`

### CODE REVIEW

**Comment by blink1073** [comment]
cc @choldgraf @consideRatio

File: github_activity/cli.py [hunk 1]
```diff
@@ -109,6 +110,12 @@
     default=None,
     help=("""The branch or reference name to filter pull requests by"""),
 )
+parser.add_argument(
+    "--all",
+    default=False,
+    action="store_true",
+    help=("""Whether to include all the GitHub tags"""),
```
Comment (choldgraf) [inline github_activity/cli.py:117]: I think we should explain a bit more what this means, as I don't think people will realize this means "generate a gigantic markdown file for every tag that exists in the repository". How about this
```suggestion
    help=("""Include all GitHub tags in the generated Changelog. Useful when generating a changelog for the entire history of a repository."""),
```

**Review by choldgraf — APPROVED** [review]
Two quick comments, but in general this looks good to me. I don't have time to do a really deep dive on the code but it seems good!

File: github_activity/github_activity.py [hunk 2]
```diff
@@ -153,6 +157,129 @@ def get_activity(
...
+    Returns
+    -------
+    entry: str
+        The markdown changelog entry for all of the release tags in the repo.
+    """
+    # Get the sha and tag name for each tag in the target repo
+    with TemporaryDirectory() as td:
+
+        subprocess.run(
+            shlex.split(f"git clone https://github.com/{target} repo"), cwd=td
```
[the remark this answers is not in the document]
Comment (blink1073) [reply github_activity/github_activity.py:214]: Yeah totally.  Thanks for the review!

**Comment by choldgraf** [comment]
Woot! And sorry for the slow turnaround on this one

**Comment by jtpio** [comment]
Any chance there could be a new release of `github-activity` that would include this change?

I tried using the `--all` to bootstrap the changelog for an existing project but it doesn't seem to be available yet.

Thanks!

**Comment by jtpio** [comment]
That was fast, thanks!

**Pull request merged**